gef-ui/server/models/application.js

```diff
@@ -82,12 +82,14 @@ const isNoticeAndCanResubmit = (application) =>
  * checks if AIN or MIN (notice)
  * checks submission count is above 0
  * checks status is changes required and if any facilities have been issued
+ * checks facilities have been completed
  */
 const applicationCanResubmitAsNotice = (application) =>
   (application.submissionType === DEAL_SUBMISSION_TYPE.AIN || application.submissionType === DEAL_SUBMISSION_TYPE.MIN) &&
   application.submissionCount > 0 &&
   [DEAL_STATUS.CHANGES_REQUIRED].includes(application.status) &&
-  facilitiesChangedToIssuedAsArray(application).length > 0;
+  facilitiesChangedToIssuedAsArray(application).length > 0 &&
+  application.facilitiesStatus.code === DEAL_STATUS.COMPLETED;
```

Comment:
is there a test file for this?

Reply:
Good spot @Zainzzkk there is a test, but it wasn't updated in the original PR, not ideal, seems the `applicationCanResubmitAsNotice` const is not tested. I think this should be done in it's own PR and pointing to `main`.

Reply:
@ttbarnes @MarRobSoftwire do you mind co-ordinating on this being tested within main as required here please? I'll resolve this thread for this PR

Reply:
I have created DTFS2-7474 for this, added to the backlog.